module.yml

```diff
@@ -103,6 +103,14 @@ Parameters:
     Default: 80
     MinValue: 1
     MaxValue: 49150
+  AppEnvironmentVariables:
+    Description: 'An optional string representation of a list of environment variables. Each key value pair is separated by ,, and each key and value by ==. eg. "key1==value1,,key2==value2"'
```

Comment:
I'm thinking if we should keep the syntax the same as in AppSecretVariables: `key=value` ? If the user has special chars, the "old" way ist still available?

Reply:
Sorry I've been slow on these. I can take a look at this tonight, I'm wondering about this one though. You're right that it is better not to have breaking changes, but wouldn't it be better for consumers not to have to use 2 different ways to add environment variables?

Reply:
For backwards compatibility you normally:

 - Introduce the new format
 - Deprecate the old one (in the docs and template)
 - After some time fully remove the old approach from the template